### DISCUSSION TITLE
How can I get last record stored in a collection?

### DISCUSSION THREAD

**Discussion by Muzammil-Bit** [isar/isar#1491] [Help] [answered]
I just want to retrieve the last record stored in the database, but there seem to be no easy way to acheive this. How can I do this?

**Comment by Jtplouffe** [comment, marked as answer]
I think the best way would be to have a field with the timestamp at which the record was created, like `DateTime createdAt`.
You could then order by this field in descending order and use the `findFirst()` function.

**Reply by Muzammil-Bit** [reply]
That seems to be a good way. Thanks a lot for the response!
